Misc/NEWS.d/next/Core and Builtins/2019-02-20-17-57-31.bpo-36052.l8lJSi.rst

```diff
@@ -0,0 +1,2 @@
+Raise a SyntaxError when we try to assign a value to `__debug__` with the
```

Comment:
I thin we don't use 'we' in the docs. Maybe:

 ```
Raise a ``SyntaxError`` when assigning a value to `__debug__` using the assignment operator.
```

Reply:
Updated